src/view/com/auth/LoggedOut.tsx

```diff
@@ -36,6 +38,12 @@ export function LoggedOut({onDismiss}: {onDismiss?: () => void}) {
     setMinimalShellMode(true)
   }, [screen, setMinimalShellMode])
 
+  React.useEffect(() => {
+    if (requestedAccountSwitchTo) {
+      setScreenState(ScreenState.S_Login)
+    }
+  }, [requestedAccountSwitchTo])
```

Comment:
Can't just set this in the useState? Or does that break back button functionality 🤔

Reply:
As much as possible, we shouldn't have effects that update state. It means something else in the logic is not quite right.